esm-samples/jsapi-deno/webmap.ts

```diff
@@ -1,7 +1,7 @@
 // @deno-types="npm:@arcgis/core/interfaces.d.ts"
-import { Application, Context } from "https://deno.land/x/oak@v11.1.0/mod.ts";
-import config from "npm:@arcgis/core/config.js";
-import WebMap from "npm:@arcgis/core/WebMap.js";
+import { Application, Context } from "https://deno.land/x/oak@v12.5.0/mod.ts";
+import config from "npm:@arcgis/core@4.26/config.js";
+import WebMap from "npm:@arcgis/core@4.26/WebMap.js";
```

Comment:
```suggestion
import config from "npm:@arcgis/core@4.27/config.js";
import WebMap from "npm:@arcgis/core@4.27/WebMap.js";
```

Reply:
Yep, thanks I was too fast on the trigger. `WebMap` updated to 4.27.